certbot/util.py

```diff
@@ -482,6 +482,8 @@ def enforce_domain_sanity(domain):
     if len(domain) > 255:
         raise errors.ConfigurationError("{0} it is too long.".format(msg))
     labels = domain.split('.')
+    if len(labels) < 2:
+        raise errors.ConfigurationError("{0} it does not have enough labels.".format(msg))
```

Comment:
Coverage decreased 0.005% because this line apparently is not covered
```
+        raise errors.ConfigurationError("{0} it does not have enough labels.".format(msg))
```

However for `test_localhost` to pass, an exception _must_ be thrown from this line, as running `test_localhost` on `master` (i.e. c00568a) without this patch results in a test failure.  

Any ideas what's going on here?